build-system/tasks/css/index.js

```diff
@@ -74,6 +74,13 @@ const cssEntryPoints = [
     outCss: 'amp-story-player-iframe-v0.css',
     append: false,
   },
+  {
+    // Internal CSS used for the desktop panels experiment iframes inside `src/amp-story-player/amp-story-player.js`.
+    path: 'amp-story-player-iframe-desktop-panels.css',
+    outJs: 'amp-story-player-iframe-desktop-panels.css.js',
+    outCss: 'amp-story-player-iframe-desktop-panels-v0.css',
+    append: false,
+  },
```

Comment:
Let's just do a `@import` in the main player CSS so we can remove these build targets / all the javascript to inject the CSS.
We'll also need to make sure our code fails correctly for publishers who don't import the CSS